### DISCUSSION TITLE
Teleport SSH client session timeout (default 1minute)

### DISCUSSION THREAD

**Discussion by ghost** [gravitational/teleport#35359] [Q&A] [answered]
Hey, I could use a little help that I couldn't find on the forum (sorry if I missed it, admin!). How do I set up the SSH idle session timeout for my clients? I've got Ubuntu Linux servers on AWS, and despite checking forums and docs, nothing I've tried so far has given me the expected results. Any ideas? Thanks!

**Comment by zmb3** [comment, marked as answer]
You're probably looking for `client_idle_timeout` as specified in https://goteleport.com/docs/reference/config/#reference-configurations

**Reply by ghost** [reply]
I discovered that in my case it was related to the loadbalancer, from my infrastructure and not a configuration in Teleport.
